shop_richcatalog/migrations/0001_initial.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
```

Comment:
This PR seems to be missing the original contents (South) of this file.  Just seems odd.  The branch looks okay.